Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -519,126 +568,34 @@ internal void OnStartServer()
             // add to spawned (note: the original EnableIsServer isn't needed
             // because we already set m_isServer=true above)
             server.spawned[netId] = this;
-
-            foreach (NetworkBehaviour comp in NetworkBehaviours)
-            {
-                // an exception in OnStartServer should be caught, so that one
-                // component's exception doesn't stop all other components from
-                // being initialized
-                // => this is what Unity does for Start() etc. too.
-                //    one exception doesn't stop all the other Start() calls!
-                try
-                {
-                    comp.OnStartServer();
-                }
-                catch (Exception e)
-                {
-                    Debug.LogError("Exception in OnStartServer:" + e.Message + " " + e.StackTrace);
-                }
-            }
         }
 
         bool clientStarted;
-        internal void OnStartClient()
+        internal void StartClient()
         {
             if (clientStarted)
                 return;
             clientStarted = true;
-
-            if (LogFilter.Debug) Debug.Log("OnStartClient " + gameObject + " netId:" + netId);
-            foreach (NetworkBehaviour comp in NetworkBehaviours)
-            {
-                // an exception in OnStartClient should be caught, so that one
-                // component's exception doesn't stop all other components from
-                // being initialized
-                // => this is what Unity does for Start() etc. too.
-                //    one exception doesn't stop all the other Start() calls!
-                try
-                {
-                    comp.OnStartClient(); // user implemented startup
-                }
-                catch (Exception e)
-                {
-                    Debug.LogError("Exception in OnStartClient:" + e.Message + " " + e.StackTrace);
-                }
-            }
         }
 
         static NetworkIdentity previousLocalPlayer = null;
-        internal void OnStartLocalPlayer()
+        internal void StartLocalPlayer()
```

Comment:
Is this called anywhere?

Reply:
Added as a listener in Awake and called in NetworkClient.CheckForLocalPlayer